src/test/resources/import/import_zip_preview.json

```diff
@@ -463,11 +464,10 @@
             "maxAcceptedUsers": 15
           }
         },
-        "adapter": {
-          "name": "adapter1",
-          "baseEndpoint": "http://endpoint1/",
-          "description": "test adapter",
-          "models": []
+        "source": {
```

Comment:
how does it works? looks like invalid json. "source" is top level preview object?